ai2thor/video_controller.py

```diff
@@ -0,0 +1,336 @@
+"""
+A video controller for ai2thor
+Basic example:
+from ai2thor.controller import VideoController
+with VideoController() as vc:
+    vc.transform(vc.moveAhead())
+    vc.exportVideo('thor.mp4')
+Known issues:
+- Multi agent rotations don't work (since TeleportFull breaks when passing in an AgentID)
+"""
+
+from ai2thor.controller import Controller
+import cv2
```

Comment:
Can you add opencv-python to setup.py? Do you know if you need a particular version?